cime_config/config_archive.xml

```diff
@@ -29,7 +29,19 @@
     <rest_file_extension>r\.(INSTANT|AVERAGE|MAX|MIN)\.n(step|sec|min|hour|day|month|year)s_x\d*</rest_file_extension>
     <rest_file_extension>rhist\.(INSTANT|AVERAGE|MAX|MIN)\.n(step|sec|min|hour|day|month|year)s_x\d*</rest_file_extension>
     <!-- The following matches "hi.AVGTYPE.FREQUNITS_xFREQ.TIMESTAMP.nc"-->
+    <!-- what's the "hi" for? I don't think we follow that convention (maybe we should?) -->
     <hist_file_extension>hi\.(INSTANT|AVERAGE|MAX|MIN)\.n(step|sec|min|hour|day|month|year)s_x\d*\.\d{4}-\d{2}-\d{2}-\d{5}\.nc$</hist_file_extension>
```

Comment:
I think "hi" is mean to refer to E3SM's "h for history" followed by "i" for index, which can be h0, h1, etc. If I recall correctly, coupler history files literally use the name "hi" because there's only 1 of them. We specifically chose *not* to follow the "hi" formatting for EAMxx output names because it is cryptic and led to a limit on how many output files we could have. So I'm strongly opposed to hist file extension "hi" unless it refers to other components than EAMxx.

Reply:
I think the "i" for EAM is "initial".  Its a snapshot of the initial condition dumped at the start of a run.  But in the coupler "i" is instantaneous.   EAM outputs a file with "hi" in the name along with h0, h1, h2, etc.

Reply:
oh yeah, we called our scream initial conditions screami files because we thought it was cute.

Reply:
Here is where Jim's knowledge is important. But trying to remember, I think we had to add something, otherwise rest files were also picked up as hist files. I'm not 100% sure about that though, we should check (and ask Jim!).

Reply:
Everything looks ok to me except maybe this second `hist_file_extension`. It looks exactly the same as the one above except without the leading `hi`, this means the files you want to match are identical to the ones it was already matching except with no `hi` in the file name. Is that what is intended here?